taxi/admin.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    pass

```

Comment:
U can just use `admin.site.register(Manufacturer)`. You don't make any changes here, so no need to create a class